tests/jballerina-unit-test/src/test/resources/test-src/record/open_record_type_inclusion.bal

```diff
@@ -198,6 +198,26 @@ function testCreatingRecordWithOverriddenFields() {
     assertEquality("UNKNOWN", dummyPerson.name);
 }
 
+public type Foo record {
+    anydata body;
+};
+
+// Out of order inclusion test : added to test a NPE
+type Bar record {
+    *Foo;
+    Baz body;   // defined after the type definition
```

Comment:
Let's add a negative test where `Baz` is not a subtype of anydata too.

Reply:
Will send a separate PR for this.

Reply:
Done 018e39750fef025355f91e69905e7b5bc00d6af1